src/web/src/components/Posts/Post.tsx

```diff
@@ -374,6 +382,11 @@ const PostComponent = ({ postUrl, currentPost, totalPosts }: Props) => {
         </ListSubheader>
       )}
       <div className={classes.content}>
+        {isMedia && (
+          // a 'guid' from a YouTube video is usually written as 'yt:video:id'
+          <YouTube videoId={post.guid.split(':')[2]} className={classes.video} />
```

Comment:
Question: what's the difference between the `guid` and `url` for  a YouTube video?

Reply:
Also, let's make a function that does the id extraction, so you can name it `extractId` or something so this is self documenting.

Reply:
The `url` already has the link to the youtube video, so there is no need to extract the video `id` from the `guid`.